articles/quickstarts/search.md

```diff
@@ -114,10 +106,11 @@ If you enjoyed this quickstart, check out some of the resources below to learn m
 - [Back to the Getting Started with QDK guide](xref:microsoft.quantum.welcome)
 - Try a more general Grover's search algorithm [sample](https://github.com/microsoft/Quantum/tree/master/samples/algorithms/database-search)
 - [Learn more about Grover's search with the Quantum Katas](xref:microsoft.quantum.overview.katas)
-- Read more about [Amplitude amplification](xref:microsoft.quantum.libraries.standard.algorithms#amplitude-amplification), the quantum computing technique behind Grover's search algorithm
+- Read more about [Amplitude amplification][amplitude-amplification], the quantum computing technique behind Grover's search algorithm
```

Comment:
Why was the link removed? In the rich view this doesn't show as a clickable link.

Reply:
The link was moved to use footnote notation so as to suppress a warning for using fragment identifiers and xrefs together. In previous renderings, the link still worked with no change in target, so it sounds like there was a regression. Let me investigate.

Reply:
If you look at this file in the [live preview of this PR](https://review.docs.microsoft.com/en-us/quantum/quickstarts/search?branch=pr-en-us-804), the link works correctly.

Reply:
Resolution: the link works on the internal preview site and goes to the correct destination, but it does not work correctly in GitHub's built-in rendering.